config

```diff
@@ -13,7 +13,19 @@ ngx_module_srcs=" \
                 $ngx_addon_dir/src/ssl/ngx_lua_kong_ssl.c \
                 "
 
-ngx_module_incs="$ngx_addon_dir/src"
+if [ -z "$NGX_LUA_LOC" ]; then
+    ngx_lua_loc="../ngx_lua-*"
+else
+    ngx_lua_loc="$NGX_LUA_LOC"
+fi
+
+if [ -z "$NGX_STREAM_LUA_LOC" ]; then
```

Comment:
I am not sure why we need this line, but I want to mention that the stream module config is https://github.com/Kong/lua-kong-nginx-module/blob/master/stream/config, it seems we do not need to specify stream module-related config in  HTTP module config.

Reply:
yeah, my bad. It has been fixed. Thanks a lot.